src/Core/Components/Dialog/Parameters/DialogParameters.cs

```diff
@@ -11,7 +11,22 @@ public class DialogParameters : ComponentParameters, IDialogParameters
     /// left (full height), right (full height)
     /// or screen middle (using Width and Height properties).
     /// </summary>
-    public virtual HorizontalAlignment Alignment { get; set; } = HorizontalAlignment.Center;
+    public virtual HorizontalAlignment Alignment
+    {
+        set
+        {
+            if (value == HorizontalAlignment.Stretch)
+            {
+                throw new ArgumentException("Alignment HorizontalAlignment.Stretch is not supported for DialogParameters");
+            }
+            _alignment = value;
+        }
+        get
+        {
+            return _alignment;
+        }
+    }
+    private HorizontalAlignment _alignment = HorizontalAlignment.Center;
```

Comment:
Can you move the `private` variable to the top of the file (just below `class`)? To be closer to our ‘Naming Conventions’.